src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1720,12 +1720,52 @@
   <AbstractNode Name="PatternSyntax" Base="CSharpSyntaxNode" />
   <Node Name="DeclarationPatternSyntax" Base="PatternSyntax">
     <Kind Name="DeclarationPattern" />
-    <Field Name="Type" Type="TypeSyntax" />
+    <Field Name="Type" Type="TypeSyntax"/>
     <Field Name="Designation" Type="VariableDesignationSyntax">
       <Kind Name="SingleVariableDesignation"/>
       <Kind Name="DiscardDesignation"/>
     </Field>
   </Node>
+  <Node Name="DeconstructionPatternSyntax" Base="PatternSyntax">
```

Comment:
>DeconstructionPatternSyntax [](start = 14, length = 27)

It doesn't look like Deconstruction Pattern is mentioned in [https://github.com/dotnet/csharplang/blob/master/proposals/patterns.md](https://github.com/dotnet/csharplang/blob/master/proposals/patterns.md). Is there a different document with specification for it?

Reply:
This is not tracking the old proposed spec written during the C# 7 timeframe. LDM issues for this iteration are tracked at dotnet/csharplang#1054